politico_api/.coverage

```diff
@@ -0,0 +1 @@
tico_api\\v1\\models\\office.py":[1,2,3,5,7,11,13,17,25,31,36,38,41,44,47,33,27,14,15,18,19,20,21],"C:\\Users\\User\\Desktop\\POLITICO-API\\politico_api\\v1\\models\\model_functions.py":[1,3,7,11,20,28,34,44,55,70,30,13,14,16,57,58,59,22,23,61,62,63,72,73,74],"C:\\Users\\User\\Desktop\\POLITICO-API\\politico_api\\v1\\views\\api_functions.py":[1,3,7,16,23,32,41,51,62,72,80,88,64,65,66,74,75,18,19,20,21,67,68,9,10,11,12,34,35,37,43,44,47,45,36,82,84,83,53,55,56,57],"C:\\Users\\User\\Desktop\\POLITICO-API\\politico_api\\v1\\views\\api_response_data.py":[5,6,10,11,12,13,14,17,18,19,20,21,22,23,30,31,32,36,37,38,42,43,44,45,48,49,50,53,54,57],"C:\\Users\\User\\Desktop\\POLITICO-API\\politico_api\\v1\\views\\party\\__init__.py":[1],"C:\\Users\\User\\Desktop\\POLITICO-API\\politico_api\\v1\\views\\party\\party_blueprint.py":[1,2,3,4,5,7,9,13,46,70,79,49,50,51,54,57,61,62,64,65,58,55,16,17,19,22,23,39,41,42,26,30,31,81,84,86,89,93,97,98,99,90,94],"C:\\Users\\User\\Desktop\\POLITICO-API\\politico_api\\v1\\models\\party.py":[1,2,3,5,7,14,16,23,32,36,40,44,48,50,53,56,46,17,18,19,20,21,25,27,28,29],"C:\\Users\\User\\Desktop\\POLITICO-API\\politico_api\\v2\\views\\__init__.py":[1],"C:\\Users\\User\\Desktop\\POLITICO-API\\politico_api\\v2\\views\\users\\users_blueprint.py":[1,2,4,8,12,13,14,17,19,20,21,26,27,28,29,30,32,33,34,35,36,37,22,23,24,25],"C:\\Users\\User\\Desktop\\POLITICO-API\\politico_api\\v2\\models\\user.py":[1,2,4,6,8,13,15,16,10,11],"C:\\Users\\User\\Desktop\\POLITICO-API\\politico_api\\v2\\models\\DBConnections\\UserConnectDb.py":[1,3,5,11,27,34,53,74,6,7,75,76,13,14,15,17,18,78,79,80,84,85,86,88,89,29,30,31,32,91,54,55,56,35,36,38,39,41,42,44,59,60,62,63,64,65],"C:\\Users\\User\\Desktop\\POLITICO-API\\politico_api\\tests\\test_office_post.py":[2,3,4,5,7,9,13,14,20,30,38,44,46,51,60,69,75,15,39,40,41,42,31,32,33,34,21,22,23,25,47,52,53,54,56,61,62,63,65,70,71,72,73],"C:\\Users\\User\\Desktop\\POLITICO-API\\politico_api\\tests\\test_party_delete.py":[1,2,3,4,6,7,9,11,17,29,40,51,12,41,43,44,46,47,30,32,33,35,36,18,20,21,23,24,52,54,55,57,58],"C:\\Users\\User\\Desktop\\POLITICO-API\\politico_api\\tests\\functions_for_tests.py":[1,4,5,6],"C:\\Users\\User\\Desktop\\POLITICO-API\\politico_api\\tests\\test_party_patch.py":[2,3,4,5,7,8,10,12,17,29,34,36,41,13,18,20,21,23,24,30,32,37,42,45],"C:\\Users\\User\\Desktop\\POLITICO-API\\politico_api\\tests\\test_party_post.py":[2,3,4,5,7,8,11,17,19,24,37,49,61,74,87,96,101,103,109,122,135,147,159,171,180,20,88,89,93,38,39,40,42,43,44,45,50,51,54,55,56,57,62,63,67,68,69,70,75,76,80,81,82,83,25,26,27,28,30,31,32,33,104,136,137,140,141,142,143,123,124,127,128,129,130,160,161,164,165,166,167,148,149,152,153,154,155,110,111,114,115,116,117,172,173,176,177],"C:\\Users\\User\\Desktop\\POLITICO-API\\politico_api\\tests\\test_user_v2_post.py":[1,2,3,5,6,7,9,15,17,23,34,45,55,63,67,69,73,82,91,100,108,110,114,18,35,36,39,40,46,47,50,51,24,25,28,29,56,57,60,61,70,101,102,105,106,74,75,78,79,92,93,96,97,83,84,87,88]}}
```

Comment:
You should add this file to the .gitginore

Reply:
I have altered the .gitignore to ignore .coverage. I hope it is okay now